### PR TITLE
Add Envoy support page and update theme

### PR DESCRIPTION
**Pull request by daniellemaxwell**
Scope of Changes:
This PR adds a page providing details about Envoy support. The Hugo theme was also updated which required a few build errors to be resolved.

Follow on stories were created to continue investigating some warnings that appear when building the site: SC-28640 & SC-28641

Acceptance Criteria:
https://tinyurl.com/2982xtyr

### CODE REVIEW

File: docs/.gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 .hugo_build.lock
+.hvm
```
Comment (daniellemaxwell) [inline docs/.gitignore:2]: I installed the hugo version manager to see if that would help me resolve issues with the links not working.

File: docs/hugo.toml [hunk 2]
```diff
@@ -14,7 +14,6 @@ enableMissingTranslationPlaceholders = false
 [params]
 
     # Source Code repository section
-    description = "TRISA Documentation"
```
Comment (daniellemaxwell) [inline docs/hugo.toml:17]: This was removed to resolve a build error. An update to the theme required moving the description to `_index.en.md`. This was previously done, so deleting this line was the only requirement.

File: docs/content/envoy/api.en.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ weight: 90
 ---
 
 {{% notice style="tip" title="API Credentials" icon="rocket" %}}
-In order to use the API you will have to generate an API Key Client ID and Secret either from the user interface, or by using the command line tool as described in ["creating api keys"]({{< relref "deploy.md#creating-api-keys" >}}).
+In order to use the API you will have to generate an API Key Client ID and Secret either from the user interface, or by using the command line tool as described in ["creating api keys"]({{% relref "deploy.md#creating-api-keys" %}}).
```
Comment (daniellemaxwell) [inline docs/content/envoy/api.en.md:10]: The change was made after noticing a warning message.

File: docs/content/envoy/support.end.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+date: 2024-12-10T13:53:33-05:00
+lastmod: 2024-12-10T13:53:33-05:00
+description: "Envoy Support"
+weight: 100
+---
+
+## How can we help?
+
+
+|                        | Open Source            | One-Time Setup         | Managed Service |
```
Comment (daniellemaxwell) [inline docs/content/envoy/support.end.md:12]: This table comes from the Envoy repo's README. If additional information should be included, let me know.

**Review by pdamodaran — APPROVED** [review]
Looks good @daniellemaxwell!  Thanks for updating the TRISA documentation!

**Pull request merged**